### ISSUE TITLE
Species Code List

### ISSUE THREAD

**Issue by mkm1879** [tracefirst/usaha_committee#42]
I believe we really need to expand the species list to include at least the full range of USDA three letter codes.  

We are working on using the standard to file CVIs in USAHERDS.  It is clear that the standard list will never include all the subdivisions that an application might support but it should at least have a parent of any given group. If I remember the discussion correctly, we had planned to revisit this list at some time anyway and just never got to it.

**Comment by mkm1879** [comment]
PGN Pigeon just bit me in the #$@@.  We get a lot of CVIs for racing pigeons going out of state to start racing back.  I've added it to my "relaxed" schema.
This points to an issue with our terminology.  Even though traceability doesn't really care about pigeons, traceability does care about getting the standard widely used.  And users have to process even the weird species.  Even though we provide "Other" that doesn't satisfy the use case where we really do need to communicate the weird species (in this case from Civet to USAHERDS using a list pulled from USAHERDS but going through the standard.)

**Comment by StaceySchwabenlander** [comment]
You'll have to forgive me, but I'm trying to understand how to read the [#42] Replaced SpeciesCodeType list with new values - the pink information is not the species list, it is the information in green that is the species list?  Am I understanding this correctly?  If so, this doesn't seem to match the information published on the USAHA site:  http://www.usaha.org/Committees/AnimalHealthSurveillanceandInformation/AHSISDataStandards.aspx

The species list in that document seems to match the area in pink, not green.  Can anyone clarify?  Thanks!

**Comment by mkm1879** [comment]
I agree, it looks like the Word document on the USAHA site was made from an older copy of the draft.
This raises the larger issue of keeping the standard current with the committee's work during the DSTU period.  Should the "official" version be frozen until some date when all agreed upon updates are made?  Or should we ballot interim point version upgrades?

**Comment by mkm1879** [comment]
One more weird species has reared its head here in SC.  Much of our NPIP paperwork arrives with just POU-Poultry.  That has been a valid "species" in USDA databases, etc.  I'm "hacking" that into our local work.  Do we want it in the standard when we do the next round?  We have the even more generic AVI-Avian.

**Comment by mkm1879** [comment]
Me again.  There is a huge potential list of minor species that can legitimately be included on CVIs.  The ones that reared their heads here recently are all the game bird species.  I think I've listed commercial pigeons earlier.  (Not a minor species to our commercial producer here in SC.)  I think we need to think about how we support those while avoiding an explosion of codes.
